tests/integration_tests/functional/test_cpu_features_host_vs_guest.py

```diff
@@ -82,93 +168,28 @@ def test_host_vs_guest_cpu_features(uvm_nano):
 
     match CPU_MODEL:
         case CpuModel.AMD_MILAN:
-            host_guest_diff_5_10 = {
-                "amd_ppin",
-                "aperfmperf",
-                "bpext",
-                "cat_l3",
-                "cdp_l3",
-                "cpb",
-                "cqm",
-                "cqm_llc",
-                "cqm_mbm_local",
-                "cqm_mbm_total",
-                "cqm_occup_llc",
-                "decodeassists",
-                "extapic",
-                "extd_apicid",
-                "flushbyasid",
-                "hw_pstate",
-                "ibs",
-                "irperf",
-                "lbrv",
-                "mba",
-                "monitor",
-                "mwaitx",
-                "overflow_recov",
-                "pausefilter",
-                "perfctr_llc",
-                "perfctr_nb",
-                "pfthreshold",
-                "rdpru",
-                "rdt_a",
-                "sev",
-                "sev_es",
-                "skinit",
-                "smca",
-                "sme",
-                "succor",
-                "svm_lock",
-                "tce",
-                "tsc_scale",
-                "v_vmsave_vmload",
-                "vgif",
-                "vmcb_clean",
-                "wdt",
-            }
-
-            host_guest_diff_6_1 = host_guest_diff_5_10 - {
-                "lbrv",
-                "pausefilter",
-                "pfthreshold",
-                "sme",
-                "tsc_scale",
-                "v_vmsave_vmload",
-                "vgif",
-                "vmcb_clean",
-            } | {"brs", "rapl", "v_spec_ctrl"}
-
             if global_props.host_linux_version_tpl < (6, 1):
-                assert host_feats - guest_feats == host_guest_diff_5_10
+                assert host_feats - guest_feats == AMD_MILAN_HOST_ONLY_FEATS
```

Comment:
is there a reason why we're dropping the 5_10 suffixes?

Reply:
I was thinking that 5_10 is the baseline, so does not need a suffix. Also for consistency since the previous  global `INTEL_HOST_ONLY_FEATS` did not include kernel version.